VRCFaceTracking/OSC/OSCParams.cs

```diff
@@ -147,13 +151,15 @@ public override void ResetParam(ConfigParser.Parameter[] newParams)
                 _negativeParam.ResetParam(newParams);
 
                 // Get all parameters starting with this parameter's name, and of type bool
-                var boolParams = newParams.Where(p => p.input.Type == typeof(bool) && p.name.StartsWith(_paramName));
+                Regex regex = new Regex(@"(?<!(v\d+))/" + _paramName + @"\d+$|^" + _paramName + @"\d+$");
+
+                var boolParams = newParams.Where(p => 
+                    p.input.Type == typeof(bool) && regex.IsMatch(p.name));
 
                 var paramsToCreate = new Dictionary<string, int>();
                 foreach (var param in boolParams)
                 {
-                    // Cut the parameter name to get the index
-                    if (!int.TryParse(param.name.Substring(_paramName.Length), out var index)) continue;
+                    if (!int.TryParse(String.Concat(param.name.ToArray().Reverse().TakeWhile(char.IsNumber).Reverse()), out var index)) continue;
```

Comment:
What's are some good test inputs for this? There might be a simpler way to do it but I want to see what it should work on

Reply:
That part specifically is for parsing out the int from a binary parameter (eg. v2/JawOpen4, v2/JawOpen2, v2/JawOpen1...), it basically counts from end to start and gets the first number it encounters and reverses it back. So v2/JawOpen16 would give int 16 for example. This is to work around the possibility that a loaded param won't strictly contain the base param name (eg. JawOpen) from the regex parsing (which could be like v2/JawOpen).

Basically any sets of binary params would be great to test, even testing out different nested patterns as well, like: 
```
gamer/gunk/v2/JawOpen16
test/v2/JawOpen8
123123123/v2/JawOpen4
w/t/f/v2/JawOpen2
v2/JawOpen1
```